packages/app-project/components/ClassifyPage/components/ClassifyBox/README.md

```diff
@@ -0,0 +1,3 @@
+# ClassifyBox
+
+A common box component for widgets on the Classify page.
```

Comment:
Is this gonna handle some theming or something else?

Reply:
There's going to be props for title and links (which are used in the other boxes)